GVFS/GVFS.Mount/InProcessMount.cs

```diff
@@ -29,7 +29,8 @@ public class InProcessMount
         private FileSystemCallbacks fileSystemCallbacks;
         private GVFSEnlistment enlistment;
         private ITracer tracer;
-        private BackgroundPrefetcher prefetcher;
+        private GitCleanupQueue cleanupQueue;
+        private GitCleanupScheduler cleanupScheduler;

```

Comment:
I'm wondering if we can encapsulate the queue in the scheduler so it will never be exposed and have a method on the scheduler to add a one time step instead of recurring?

Reply:
same here like a cleanupScheduler. ScheduleOneTimeCleanupStep